tests/testing_utils.py

```diff
@@ -25,6 +26,8 @@ def require_torch_gpu(test_case):
     """
     if not torch.cuda.is_available():
         return unittest.skip("test requires GPU")(test_case)
+    if not is_xpu_available():
+        return unittest.skip("test requires XPU")(test_case)
```

Comment:
```suggestion
```
Do we really need this? can't you just add a new decorator `require_torch_xpu` ? Also I am not sure the GPU tests will pass on XPUs (especially 8bit / 4bit tests) so maybe let's just remove the modifs on the slow tests and keep only the modifs on `testing_common.py`

Reply:
Yes for now it wont pass on 4/8 bits but we are in progress to support xpu kernels for the same. I will add those later once finalized. Thanks